scripts/upgrade-bare-metal.sh

```
@@ -0,0 +1,116 @@
+    $SUDO chmod -R a+rX "$PREFIX/$app"
+  fi
+done
+if [[ -d forms/dist && -d "$PREFIX/forms" ]]; then
+  log "installing forms"
+  $SUDO rm -rf "$PREFIX/forms/dist"
+  $SUDO cp -r forms/dist "$PREFIX/forms/dist"
+fi
+
+log "restarting backend"
+$SUDO systemctl restart warmbly-backend
+for i in $(seq 1 60); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)